Userland/Services/WebContent/ConsoleGlobalObject.cpp

```diff
@@ -118,4 +123,58 @@ JS_DEFINE_NATIVE_FUNCTION(ConsoleGlobalObject::dollar_underscore_getter)
     return console_global_object->m_most_recent_result;
 }
 
+JS_DEFINE_NATIVE_FUNCTION(ConsoleGlobalObject::dollar_function)
+{
+    auto* console_global_object = TRY(get_console(vm));
+    auto& window = *console_global_object->m_window_object;
+
+    auto selector = TRY(vm.argument(0).to_string(vm));
+
+    if (vm.argument_count() > 1) {
+        auto element_value = vm.argument(1);
+        if (!(element_value.is_object() && is<Web::DOM::ParentNode>(element_value.as_object()))) {
+            return vm.throw_completion<JS::TypeError>(JS::ErrorType::NotAnObjectOfType, "DOM::ParentNode");
```

Comment:
Don't refer to C++ type names in user facing errors

Reply:
Sure. I've changed these to `Node` since that seems to be the JS DOM class that corresponds best to ParentNode.